ssl/ssl_lib.c

```diff
@@ -5140,6 +5140,9 @@ int SSL_client_hello_get1_extensions_present(SSL *s, int **out, size_t *outlen)
         if (ext->present)
             num++;
     }
+    if (num == 0) {
+        return 0;
```

Comment:
I wonder whether `return 1` might be more appropriate (with `*outlen` set to zero and `*out` set to NULL). This isn't an error as such - there just aren't any extensions. I'd also recommend a documentation update to clarify what happens in the event of no extensions.